### PR TITLE
Docs: New whitelabel settings

### PR DESCRIPTION
**Pull request by tskarhed**
Docs for new whitelabel settings

### CODE REVIEW

File: docs/sources/enterprise/white-labeling.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ You can change the following elements:
 - Side menu top logo
 - Footer and help menu links
 - Fav icon (shown in browser tab)
+  > Only available in Grafana Enterprise v7.0+
```
Comment (oddlittlebird) [inline docs/sources/enterprise/white-labeling.md:29]: The way this is formatted, it is unclear if the note applies to Fav icon or to the items below. I recommend putting the (Grafana 7.0+) after each of the three new bulets.

File: docs/sources/enterprise/white-labeling.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ You can change the following elements:
...
+- Login title (Will not appear if a login logo is set)
```
Comment (oddlittlebird) [inline docs/sources/enterprise/white-labeling.md:30]: Change all the "Will"s in parenthesis to lowercase

File: docs/sources/enterprise/white-labeling.md [hunk 3]
```diff
@@ -39,13 +43,22 @@ The configuration file in Grafana Enterprise contains the following options. Eac
...
+# Set to main title on the login page (Will not appear if a login logo is set)
+;login_title =
+
+# Set to login subtitle (Will not appear if a login logo is set)
+;login_subtitle =
+
 # Set to complete URL to override login logo
 ;login_logo =
 
 # Set to complete css background expression to override login background
```
Comment (oddlittlebird) [inline docs/sources/enterprise/white-labeling.md:55]:
```suggestion
# Set to complete CSS background expression to override login background
```

File: docs/sources/enterprise/white-labeling.md [hunk 4]
```diff
@@ -39,13 +43,22 @@ The configuration file in Grafana Enterprise contains the following options. Eac
...
 # example: login_background = url(http://www.bhmpics.com/wallpapers/starfield-1920x1080.jpg)
 ;login_background =
 
+# Set to complete css background expression to override login box background
```
Comment (oddlittlebird) [inline docs/sources/enterprise/white-labeling.md:59]:
```suggestion
# Set to complete CSS background expression to override login box background
```

**Review by oddlittlebird — APPROVED** [review]
Minor edits, then this is good to go.

File: docs/sources/enterprise/white-labeling.md [hunk 5]
```diff
@@ -26,6 +26,9 @@ You can change the following elements:
 - Side menu top logo
 - Footer and help menu links
 - Fav icon (shown in browser tab)
+- Login title (will not appear if a login logo is set, Grafana v7.0+)
+- Login subtitle (will not appear if a login logo is set, Grafana v7.0+)
```
Comment (tskarhed) [inline docs/sources/enterprise/white-labeling.md:30]: Not sure about adding Grafana v7.0+ in this way. Can I get a yay or nay?

**Pull request merged**